react-rendering-lists/my-app/src/RenderList.jsx

```diff
@@ -0,0 +1,19 @@
+const pokedex = [
```

Comment:
The instructions tell you to define this in App and pass it as a prop. Otherwise, this is all correct.